### PR TITLE
Team team info

### PR DESCRIPTION
**Pull request by smcunning**
PR Notes - 
- Added test for team_info method
- Added team_info method
- All tests pass

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -36,13 +36,13 @@ def lowest_total_score
...
   def team_info
-    result = {}
+    result = { }
     teams.each do |team|
-      result[:team_id] = team.team_id
-      result[:franchise_id] = team.franchise_id
-      result[:team_name] = team.team_name
-      result[:abbreviation] = team.abbreviation
-      result[:link] = team.link
+      (result[:team_id]||=[]) << team.team_id
```
Comment (Stearnzy) [inline lib/stat_tracker.rb]: Great use of ||= to continue adding on elements!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -33,7 +33,21 @@ def lowest_total_score
...
-#------------LeagueStatistics
+#------------TeamStatistics
+
+  def team_info
+    result = { }
+    teams.each do |team|
+      (result[:team_id]||=[]) << team.team_id
+      (result[:franchise_id]||=[]) << team.franchise_id
+      (result[:team_name]||=[]) << team.team_name
+      (result[:abbreviation]||=[]) << team.abbreviation
```
[the remark this answers is not in the document]
Comment (taphill) [reply lib/stat_tracker.rb:44]: I agree with the hound here

File: lib/stat_tracker.rb (hunk 1 above)
Comment (taphill) [reply lib/stat_tracker.rb]: I agree with @Stearnzy! Good job!

**Comment by taphill** [comment]
Looks good! Good job!

**Pull request merged**